backend/gameplay/views.py

```
@@ -86,4 +86,12 @@ def post(self, request):
 		chess_game_model: ChessGame = ChessGame.objects.get(id=game_id)
 		move_list = chess_game_model.sync_get_move_list()
 
-		return Response(move_list, status=status.HTTP_200_OK)
\ No newline at end of file
+		return Response(move_list, status=status.HTTP_200_OK)
+	
+class GetSideToMoveView(APIView):
```

Pylint (3.3.7)

[refactor] Too few public methods (1/2)

(R0903)